src/validators/OmnichannelConfigValidator.ts

```diff
@@ -7,9 +7,13 @@ const validateOmnichannelConfig = (omnichannelConfig: OmnichannelConfig): void =
       throw new Error(`OmnichannelConfiguration not found`);
     }
 
-    const currentOmnichannelConfigParams = Object.keys(omnichannelConfig);
+    const mapEntries = new Map(Object.entries(omnichannelConfig));
+    let valueOfEntry;
+
     for (const key of requiredOmnichannelConfigParams) {
-      if (!currentOmnichannelConfigParams.includes(key)) {
+      //check for they key present in the map, if not present or value is undefined then throw error
+      valueOfEntry = mapEntries.get(key);
+      if (!valueOfEntry || valueOfEntry.trim() === "") {
```

Comment:
Do we need to do any type checking for the value of the entry? Ie. if valueOfEntry was a false boolean we would also enter this if condition.

Reply:
types are not defined in the required map, but probably for sanity, I can remove the trim and just relay on lenght of the value,